### ISSUE TITLE
Make the test script run on Windows

### ISSUE THREAD

**Issue by jfmengels** [jfmengels/elm-review-documentation#8]
The current test script is

```bash
elm make --docs=docs.json && elm-format src/ tests/ --validate && elm-test && elm-review && npm run package-tests -s
```

But `&&` does not work on Windows, so this prevents Windows users from helping out on this package.

I know of [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) which helps make scripts run on Windows. It does require using npm scripts, so every part will need to be defined in a separate npm script. That has its benefits, but if you can come up with a different solution, let me know.

If we go with using `npm-run-all`, the task would be to
- Create a script for every part of the test script
- Add `npm-run-all` as a dependency
- Replace the test script with one like `npm-run-all make verify-format ...`
- Make sure the scripts are run sequentially. Running them in parallel would have little performance benefits but make failures quite hard to understand I think.
- See if everything

- Optional/bonus: Make the CI run the tests on Windows (and Mac?) to make sure that works. I think we'd need to change the `runs-on` field for `test` in `.github/workflows/test.yml` to `[ ubuntu-latest, windows-latest, macos-latest ]` but I haven't tried it out. More info [here](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idruns-on).

The idea is that all of this work, if proved successful, will then be the default setup for all review packages when created by `elm-review new-package`. That can be a separate PR for `jfmengels/node-elm-review`, but I think this is a good first step before generalizing.

**Comment by fredericbonnet** [comment]
Hello, I've just submitted PR #9 that splits the `test` script into separate subscripts using `npm-run-all`. I've tested it with success on my Windows machine (some subtests fail on my machine but this has nothing to do with the NPM scripts themselves since the plain command lines produce the same outcome).

I have not implemented the CI part because I have no experience with GitHub CI. I suggest you create a separate issue.

Happy Hacktoberfest!
